client/client.go

```diff
@@ -22,13 +22,13 @@ type Client struct {
 	Endpoint   string
 	Host       string
 	Token      string
-	httpClient *http.Client
+	HttpClient *http.Client
```

Comment:
You could use the address from the server rather making this public, but this seems fine too